.github/workflows/ci.yml

```diff
@@ -0,0 +1,86 @@
+name: CI
+
+# Controls when the workflow will run
+on:
+  # Triggers the workflow on push or pull request events but only for the master branch
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  build:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest 
+
+    steps:
+      - uses: actions/checkout@v2
+      
+    #to speed up the execution time of Github Actions
+      - name: Cache multiple paths  
+        uses: actions/cache@v2
+        id: yarn-cache # use this to check for `cache-hit` (`steps.yarn-cache.outputs.cache-hit != 'true'`)
+        with:
+          path: |-
+            vendor
+            node_modules
+          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: |
+                ${{ runner.os }}-yarn-
+     
+      - uses: getong/mariadb-action@v1.1  
+        with:
+          host port: 3306 
+          mariadb version: '10.5' 
+          mysql database: 'test' 
+      
+      - uses: ruby/setup-ruby@v1    
+        with:
+           ruby-version: 2.7.1
+           bundler-cache: true
+
+           
+      - name: Setup Node
+        uses: actions/setup-node@v1
+               
+      - run: 'export PATH=$PATH:`yarn global bin`'
+      
+      - run: export DB=test
```

Comment:
This shouldn't be necessary, since the database name is coming from database.yml via the `db:create` command later on.

Reply:
ohh yeah absolutely , will be removing these two also.